poisson_lib.py

```
@@ -128,647 +163,625 @@ def analyze_poisson_outcome(
+                    <b>Visual Indicators:</b> 📊 Categorical (vs Reference) | 📉 Linear (Per-unit increase)
+                    {model_fit_html}
+                    {interaction_info}
+                </div>
             </div>
-        </div>
-    </div><br>"""
-    
-    return html_table, irr_results, airr_results, interaction_results
+        </div><br>"""
+        
+        return html_table, irr_results, airr_results, interaction_results
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)